framework/uicomponents/qml/Muse/UiComponents/internal/popupviewclosecontroller.cpp

```
@@ -137,11 +137,23 @@ void PopupViewCloseController::onApplicationStateChanged(Qt::ApplicationState st
-    if (m_isCloseOnPressOutsideParent) {
-        if (!isMouseWithinBoundaries(mousePos)) {
-            notifyAboutClose();
+    if (!m_isCloseOnPressOutsideParent || isMouseWithinBoundaries(mousePos)) {
+        return;
+    }
+
+    // Go through the transient ancestors - if any of them contain the mouse then we won't
+    // force close (i.e. we'll close this popup, but we won't close all ancestors)...
+    const QWindow* ancestor = parentWindow();
+    while (ancestor && ancestor->transientParent()) {
+        const QRectF ancestorRect = ancestor->geometry();
```

Clang (14.0.6)

[warning] variable 'ancestorRect' is not initialized

(cppcoreguidelines-init-variables)